docs/cloud-runner/01-introduction.md

```diff
@@ -0,0 +1,70 @@
+# Introduction
+## Concept - What Does Cloud Runner Do
+
+**Cloud Runner enables you to send your project to run workflows on a Cloud Provider such as AWS or GCP, most commonly to build and test your project.** 
+
+**Primarily Cloud Runner is useful for game development because it supports large projects. Cloud Runner has first class support for the Unity game engine.**
```

Comment:
```suggestion
**Cloud Runner is especially useful for game development because it supports large projects. Cloud Runner provides first class support for the Unity game engine.**
```